### ISSUE TITLE
Need an option to influence resource ordering

### ISSUE THREAD

**Issue by JohnRudolfLewis** [Swagger2Markup/swagger2markup#205]
I would like an option to influence the ordering of resources based on the order that they exist in my swagger.

**Desired:**
From the example below, I would want /v1/zebras to appear before /v1/apples

**Actual:**
The resources are sorted alphabetically

```
convertSwagger2markup {
    swaggerInput file("swagger.yaml").toString()
    outputDir asciiDocOutputDir
    config = [
            'swagger2markup.pathsGroupedBy' : 'TAGS',
            'swagger2markup.tagOrderBy' : 'AS_IS',
            'swagger2markup.propertyOrderBy' : 'AS_IS',
            'swagger2markup.extensions.springRestDocs.snippetBaseUri': snippetsOutputDir.getAbsolutePath()]
}
```

```
swagger: '2.0'
info:
  version: 0.0.0
  title: Simple API
tags:
  - name: Zebras
  - name: Aardvarks
paths:
  /v1/zebras:
    get:
      description: I would like this to appear before Aardvarks
      tags:
        - Zebras
      responses:
        200:
          description: OK
          schema:
            $ref: "#/definitions/Zebra"

  /v1/aardvarks:
    get:
      description: I would like this to appear after zebras
      tags:
        - Aardvarks
      responses:
        200:
          description: OK
          schema:
            $ref: "#/definitions/Aardvark"

definitions:
  
 
  Aardvark:
    type: object
    allOf:
      - $ref: "#/definitions/Animal"
    properties:
      somePropertyOfAardvark:
        type: string
        description: this should be displayed in the definition of Aardvark but doesn't
      
  Zebra:
    type: object
    allOf:
      - $ref: "#/definitions/Animal"
    properties:
      somePropertyOfZebra:
        type: string
        description: this should be displayed in the definition of Zebra but doesn't
    required:
      - somePropertyOfZebra
  
  Animal:
    type: object
    properties:
      somePropertyOfAnimal:
        type: string
      "_etag":
        type: string
        readOnly: true
        description: This property is actually named _etag but shows up without the underscore. It also should show that it is a readOnly property.
```

**Comment by RobWin** [comment]
Hello, 
this is a known limitation of Guava MultiMap::hashkeys(). https://github.com/Swagger2Markup/swagger2markup/blob/master/src/main/java/io/github/swagger2markup/internal/utils/TagUtils.java#L62

I see if I can fix this.

**Comment by RobWin** [comment]
Should work in v1.1.1-SNAPSHOT

**Comment by RobWin** [comment]
Fixed in v1.1.0
